pb-rs/src/types.rs

```diff
@@ -1879,7 +1890,7 @@ fn split_package(package: &str) -> (&str, &str) {
     }
 }
 
-const MAGIC_HEADER: &'static str = "//! Automatically generated mod.rs";
+const MAGIC_HEADER: &'static str = "// Automatically generated mod.rs";
```

Comment:
This is unrelated to this PR, but was failing for me when using `include!`

Reply:
Can you rebase? I think this was already done in a previous PR.